include/curl/curl.h

```diff
@@ -294,8 +294,8 @@ typedef enum {
 struct curl_fileinfo {
   char *filename;
   curlfiletype filetype;
-  time_t time; /* always zero! */
   unsigned int perm;
+  time_t time; /* always zero! */
```

Comment:
Is this one OK @bagder @jay ? Isn't this an ABI change in a public struct?

Reply:
Good catch, I didn't see that. Correct, we must not change this!

Reply:
Yes I missed that

Reply:
Thanks, dropping that one!